Dockerfile.amd64

```diff
@@ -27,12 +24,15 @@ RUN systemctl mask \
     dev-hugepages.mount \
     sys-fs-fuse-connections.mount \
     sys-kernel-config.mount \
+
     display-manager.service \
     getty@.service \
     systemd-logind.service \
     systemd-remount-fs.service \
+
     getty.target \
-    graphical.target
+    graphical.target \
+    kmod-static-nodes.service
```

Comment:
what's that?

Reply:
No idea, they were just the defaults that balena provided in their example dockerfiles so i figured it should not hurt to copy them over.

Reply:
ok